doorway-ui-components/src/headers/DoorwayHero.scss

```diff
@@ -67,6 +67,10 @@
     @apply pr-72;
   }
 
+  @media (min-width: $screen-md) AND (max-width: $screen-lg) {
```

Comment:
Can you delete the @media (min-width: $screen-md) block above since that would never apply?